papers/source/d1664.bs

```diff
@@ -348,7 +385,25 @@ constexpr iota_view(iterator first, sentinel last);
 </ins>
 </blockquote>
 
+Add to §24.6.3 Class template `iota_view` [[range.iota.view](http://eel.is/c++draft/range.iota.view)], after clause 11 (old) / 12 (new):
 
+<blockquote>
+<ins>
+```
+constexpr friend iterator begin(iota_view v);
+```
+<p><sup>13</sup> Effects: Equivalent to: `return v.begin();`</p>
+</ins>
+</blockquote>
+
+<blockquote>
+<ins>
+```
+constexpr friend auto end(iota_view v);
+```
+<p><sup>14</sup> Effects: Equivalent to: `return v.end();`</p>
+</ins>
+</blockquote>

```

Comment:
These changes make `iota_view` model `forwarding-range`.

Reply:
Fair enough; having it be a forwarding-range would be very nice.